test/scripts/palette.r2

```diff
@@ -0,0 +1,65 @@
+ec comment rgb:c50f1f
```

Comment:
why this concrete palete and not test one of the one shipped in r2?

Reply:
it's robust against changes to the shipped palettes